config/main.py

```diff
@@ -2166,6 +2166,35 @@ def delete(ctx):
     sflow_tbl['global'].pop('agent_id')
     config_db.set_entry('SFLOW', 'global', sflow_tbl['global'])
 
+#
+# 'feature_status' command ('config feature_status name value')
+# 
+@config.command('feature_status')
+@click.pass_context
+@click.argument('name', metavar='<feature-name>', required=True)
+@click.argument('value', metavar='<enabled/disabled>', required=True)
```

Comment:
I think the argument name should be `state`, and instead of setting the metavar to `<enabled/disabled>`, you should set it to "<state>" and set `type=click.Choice(["enabled", "disabled"])` to allow the user to only enter those two options. Then you don't need the `accepted_values` variable below, either.